LeaderboardBackend/appsettings.json

```diff
@@ -15,5 +15,11 @@
     "AccountRecovery": true,
     "AccountRegistration": true,
     "Login": true
+  },
+  "Limits": {
+    "Default": {
+      "Default": 64,
+      "Max": 1024
```

Comment:
Should we use multiples of 2 or 10, actually? I'd think it'd be more natural to set them to e.g. 100 and 1000, won't it?